### PR TITLE
Br69 refactor

### PR DESCRIPTION
**Pull request by zulcss**
- Split out agent and server into seperate directories in pbench namespace.
- Fix up tox.ini a bit
- Made binaries installable via setuptools
- Refactored pbench/__init__.py into a seperate files.

### CODE REVIEW

File: pbench/cli/agent/commands/metalog.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import click
+
+from pbench.agent.task.add_metalog_option import add_metalog_option
+
+#
+# pbench cli
+#
+@click.group()
+def metalog():
```
Comment (portante) [inline pbench/cli/agent/commands/metalog.py:9]: Can you add a function description to `metalog()` describing why it is a "no-op" function?

File: agent/util-scripts/unittests [hunk 2]
```diff
@@ -315,12 +315,8 @@ declare -A tools=(
     [test-30]="pbench-collect-sysinfo"
     [test-31]="pbench-copy-results"
     [test-32]="pbench-copy-results"
-    [test-33]="pbench-move-results"
```
Comment (portante) [inline agent/util-scripts/unittests:318]: Why do we need to remove these tests?

Comment (zulcss) [reply agent/util-scripts/unittests:318]: This test has moved to python so I removed the duplicate unit test.

File: server/lib/systemd/pbench-server.service [hunk 3]
```diff
@@ -10,7 +10,7 @@ After=network.target httpd.service
 
 [Service]
 Type = simple
-ExecStart = /opt/pbench-server/utils/pbench_server.py
+ExecStart = pbench-server
```
[the remark this answers is not in the document]
Comment (zulcss) [reply server/lib/systemd/pbench-server.service:13]: Fixed

File: pbench/cli/agent/commands/results.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+@click.option("-p", "--prefix", default="", help="Specify a prefix for server upload")
+@click.option(
+    "-S", "--show-server", required=False, help="Show server",
+)
+def move(user, prefix, show_server):
+    move_results(user, prefix, show_server)
+
+
+#
+# pbench backwards compatibkility
```
[the remark this answers is not in the document]
Comment (zulcss) [reply pbench/cli/agent/commands/results.py:26]: It can be..

File: agent/tool-scripts/unittests [hunk 5]
```diff
@@ -1,6 +1,4 @@
 #!/bin/bash
-export LANG=C
-export LC_ALL=C
```
Comment (portante) [inline agent/tool-scripts/unittests:3]: Why were these removed?  We had to add them in the past in order to make sure the test output was correct across multiple platforms.

Comment (zulcss) [reply agent/tool-scripts/unittests:3]: Doh! Fixed
